laze-project.yml

```
@@ -342,6 +342,8 @@ contexts:
       PROBE_RS_CHIP: esp32c3
       PROBE_RS_PROTOCOL: jtag
       CARGO_TARGET_PREFIX: CARGO_TARGET_RISCV32IMC_UNKNOWN_NONE_ELF
+      # c3 needs a bit more isr stack.
+      isr_stacksize_required_default: "4096"
```

4096 is still not enough for the `http-client` example, 32768 isn't either, but 65536 is. It does work if we set this here, but maybe we should set that for the biggest examples only (which include `http-server` and maybe others)?

```suggestion
      isr_stacksize_required_default: "65536"
```

---

huh, I thought I checked this, was wondering why only c3 needs more. I'll do some more testing, maybe it is the wifi ISRs or stack that needs more memory, we can selectively increase that also.

---

done